diff --git a/node_modules/@gorhom/bottom-sheet/src/components/bottomSheetContainer/BottomSheetContainer.tsx b/node_modules/@gorhom/bottom-sheet/src/components/bottomSheetContainer/BottomSheetContainer.tsx
index 9d42ef1..900f5e2 100644
--- a/node_modules/@gorhom/bottom-sheet/src/components/bottomSheetContainer/BottomSheetContainer.tsx
+++ b/node_modules/@gorhom/bottom-sheet/src/components/bottomSheetContainer/BottomSheetContainer.tsx
@@ -48,6 +48,7 @@ function BottomSheetContainerComponent({
 
       containerRef.current?.measure(
         (_x, _y, _width, _height, _pageX, pageY) => {
+          if(!containerOffset?.value)return;
           containerOffset.value = {
             top: pageY,
             left: 0,
